testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -244,6 +244,21 @@
   end
 end
 
+#
+# Click on a button with a given "id"
+When(/^I click on button with id "([^"]*)"$/) do |button_id|
+  find(:xpath, ".//button[@id=\"#{button_id}\"]").click
+end
+
+#
+# Click on a text which appears inside of <div> with
+# the given "id"
+When(/^I click on "([^"]*)" text in Ansible paths$/) do |text|
+  within(:xpath, '//div[@id="ansible-path-content"]') do
+    find('h6', text: text).click
+  end
+end
+
```

Comment:
I would say we don't need this special step.

Could you instead try to use this step in your scenario?

``` #
# Click on a button which appears inside of <div> with
# the given "id"
When(/^I click on "([^"]*)" in element "([^"]*)"$/) do |text, element_id|
  within(:xpath, "//div[@id=\"#{element_id}\"]") do
    click_button_and_wait(text, match: :first)
  end
end
```

Reply:
I tried that one but it didn't work since it's expecting a button.

In this case, what we have here is not a button but a "collapsible" div that it's only containing `<h6>text</h6>` where I need to click to get the content loaded.

That's why I had to implement this particular step to deal with those "paths" boxes in the Ansible UI.